kuksa-client/kuksa_client/__main__.py

```diff
@@ -43,10 +43,16 @@
 from kuksa_client import KuksaClientThread
 from kuksa_client import _metadata
 
-DEFAULT_SERVER_ADDR = "grpc://127.0.0.1:55555"
-
 scriptDir = os.path.dirname(os.path.realpath(__file__))
 
+DEFAULT_SERVER_ADDR = os.environ.get("SERVER_ADDR", "grpc://127.0.0.1:55555")
```

Comment:
Should we try to agree on a long term (ideal) pattern. Like here:

* [Databroker](https://github.com/eclipse/kuksa.val/tree/master/kuksa_databroker) uses `KUKSA_DATA_BROKER_ADDR`
* [Dbcfeeder](https://github.com/eclipse/kuksa.val.feeders/tree/main/dbc2val) uses `KUKSA_ADDRESS`

Here we have a third option. Another topic - are we fine with names that possibly could clash with other env variables, or do we prefer to always have a `KUKSA_ `prefix?

Reply:
I like the ```KUKSA_``` option. Will change. No strong opinion on what comes after it.

Reply:
If the address can be used for both Server And Broker then I think `KUKSA_ADDRESS` is good